### DISCUSSION TITLE
Serverless function timeout when trying to sign up for plus.

### DISCUSSION THREAD

**Discussion by brettshumaker** [ariakit/ariakit#3107] [General]
👋 Not sure if this is the correct place, but I just tried to sign up for Ariakit Plus, I got the receipt email, but the screen ended with vercel saying that the serverless function has timed out.

<img width="529" alt="Screenshot 2023-11-28 at 12 56 19 PM" src="https://github.com/ariakit/ariakit/assets/1558827/5d9325e4-0217-4e0c-b5df-5771aabdf676">

I never received any login information and trying to log in with my email address says it can't find my account. Clicking on "Sign up" also end in the same time out error from Vercel. I'm sure someone is getting notified of the error, but wanted to surface it here too.

**Comment by diegohaz** [comment]
Hi @brettshumaker! I'm sorry that happened. I'm taking a look into that and will get resolved asap.

Sent you an email.
